src/app/components/details/details.rs

```diff
@@ -87,7 +87,7 @@ impl AlbumDetailsWidget {
         let is_up_to_date = widget.header_revealer.reveals_child() == visible;
         if !is_up_to_date {
             widget.header_revealer.set_reveal_child(visible);
-            widget.headerbar.set_title_visible(!visible);
+            widget.headerbar.set_title_visible(true);
```

Comment:
think the line can be removed altogether as well as the property in the ui file if we're using opacity now :)

Reply:
Yeah, this is a dirty hack - title was invisible if not explicitly set visible somehow.